app.js

```diff
@@ -10,3 +10,151 @@ document.querySelectorAll('.menu-item').forEach((n) => n.addEventListener('click
+    featured_image: 'image/styling.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/rmjspecial',
+    live_demo: 'https://github.com/rmjspecial/personal-portfolio',
+  },
+  Fitness: {
+    title: 'Fitness',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'image/fitness.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/rmjspecial',
+    live_demo: 'https://github.com/rmjspecial/personal-portfolio',
+  },
+};
+
+// reverse the order of work card
+function workCardShouldReverse(index) {
+  if (index % 2 === 0) {
+    return 'row-reverse';
+  }
+  return '';
+}
+
+// create the breadcrumbs
+function breadcrumbsHtml(workObj, key, bread = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const breadcrumb in workObj[key].breadcrumbs) {
+    if (breadcrumb) {
+      bread += `<li>${workObj[key].breadcrumbs[breadcrumb]}</li>`;
+    }
+  }
+  /* eslint-enable no-restricted-syntax */
+
+  return bread;
+}
+
+// create the language list
+function languageHtml(workObj, key, langList = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const lang in workObj[key].lang_list) {
+    if (lang) {
+      langList += `<li><span>${workObj[key].lang_list[lang]}</span></li>`;
+    }
+  }
+
+  return langList;
+}
+
+//  select the works container
+const workCardContainer = document.querySelector('.works-grid-container');
+let workCard = '';
+let index = 0;
+
+// create the work cards
+for (const key in works) {
+  if (key) {
+    index += 1;
+    workCard
+      += `
+        <div class="flex works-grid-item ${workCardShouldReverse(index)}">
+          <div class="work-image">
+            <img class="width-100" src="${works[key].featured_image}" alt="${works[key].title}">
+          </div>
+          <div class="flex work-details-wrapper">
```

Comment:
- [ ] Please increase the size of the project title and tags and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174288973-6a28645d-bc60-4e29-8ff2-3e4466ea448c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/174289053-4be25dc8-b276-4dc2-92c7-4d8a47ccdf04.png)